thirdeye/thirdeye-frontend/app/pods/manage/alert/template.hbs

```diff
@@ -73,7 +72,9 @@
         </li>
         <li class="te-search-results__row">
           <div class="te-search-results__option te-search-results__option--solo">Anomaly Pattern</div>
-          <div class="te-search-results__value-solo" title={{alertData.pattern}}>{{alertData.pattern}}</div>
+          <div class="te-search-results__value-solo" title={{alertData.pattern}}>
+            {{if alertData.pattern alertData.pattern 'N/A'}}
```

Comment:
I think you can extract `alertData.pattern` into a variable in the component, set its value there, and then reference it here to simplify template logic.

Reply:
aggreed, it can even be a cp `getWithDefault('alertData.pattern', 'N/A')` note: this only works for undefined.